addons/medical_treatment/functions/fnc_canDigGrave.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: Ruthberg, commy2, esteldunedain, drofseh
+ * Checks if a unit can dig a grave at the position of the patient.
+ *
+ * Arguments:
+ * 0: Medic (not used) <OBJECT>
+ * 1: Patient <OBJECT>
+ *
+ * Return Value:
+ * Can dig <BOOL>
+ *
+ * Example:
+ * [ACE_player] call ace_trenches_fnc_canDigGrave
+ *
+ * Public: No
+ */
+
+params ["", "_patient"];
+
+GVAR(allowGraveDigging) && {_patient call EFUNC(common,canDig)}
```

Comment:
```suggestion
GVAR(allowGraveDigging) && {_patient call EFUNC(common,canDig)} && {_medic call EFUNC(trenches,hasEntrenchingTool)}
```

Reply:
Maybe do `GVAR(allowGraveDigging) && {_patient call EFUNC(common,canDig)} && {["ace_trenches"] call EFUNC(common,isModLoaded) && {_medic call EFUNC(trenches,hasEntrenchingTool)}}`?

Reply:
If you include the suggested changes above these suggested changes, it does:

```SQF
params ["_medic", "_patient"];

if !(["ace_trenches"] call EFUNC(common,isModLoaded)) exitWith {false};

GVAR(allowGraveDigging) && {_patient call EFUNC(common,canDig)} && {_medic call EFUNC(trenches,hasEntrenchingTool)}
```

I just learnt today how to make the multiline suggestions, so next time the suggestions I'll make will be cleaner.

Reply:
Ahh, seen!